app/celery/celery_app.py

```
@@ -470,3 +472,4 @@ def cleanup_async_tasks_on_shutdown(sender, **kwargs):
 import app.celery.tasks.parsing_tasks  # noqa # Ensure the task module is imported
 import app.celery.tasks.agent_tasks  # noqa # Ensure the agent task module is imported
 import app.modules.event_bus.tasks.event_tasks  # noqa # Ensure event bus tasks are registered
+import app.celery.tasks.attachment_cleanup_tasks  # noqa # Ensure attachment cleanup tasks are registered
```

Ruff (0.14.14)

[warning] Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)